src/components/pages/Groups/AddGroupPanel/AddGroupPanel.test.tsx

```diff
@@ -125,7 +160,7 @@ test("should add entitlements", async () => {
   await userEvent.click(screen.getByRole("button", { name: "Create group" }));
   await waitFor(() => expect(patchDone).toBeTruthy());
   expect(patchResponseBody).toBe(
-    '{"patches":[{"entitlement":{"entity_type":"editors","entitlement_type":"can_read","entity_name":"client"},"op":"add"}]}',
+    '{"patches":[{"entitlement":{"entity_type":"client","entitlement_type":"can_read","entity_name":"editors"},"op":"add"}]}',
```

Comment:
The Resource type and Resource fields were reversed initially. I fixed this issue and that's why we need to change the places of the values here as well.